<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsNotebookPage</class>
 <widget class="QWidget" name="SettingsNotebookPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Appearance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSyntaxHighlighting">
     <property name="text">
      <string>Syntax Highlighting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSyntaxCompletion">
     <property name="text">
      <string>Syntax Completion</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLineNumbers">
     <property name="text">
      <string>Line Numbers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLatexTypesetting">
     <property name="text">
      <string>LaTeX Typesetting</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAnimations">
     <property name="text">
      <string>Animations</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Evaluation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkReevaluateEntries">
     <property name="text">
      <string>Re-evaluate entries automatically</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAskConfirmation">
     <property name="text">
      <string>Ask for confirmation when restarting</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>67</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
